test/brsTypes/Double.test.js

```diff
@@ -413,4 +413,39 @@ describe("Double", () => {
             expect(six.or(new Double(3.4444))).toEqual(new Int32(7));
         });
     });
+
+    describe("validate typecasting", () => {
+        // If we pass floatValue as argument for Double() thne doubleValue = 'Double { kind: 7, value: Float { kind: 6, value: 10.88889 } }'
+        // But if pass '6.2' then it = 'Double { kind: 7, value: 10.88889 }'
+        // while running in interpreter: Double, value: 10.88889
+        // on Roku:                      Double, value: 10.88888835907
+        xit("passing float", () => {
```

Comment:
Is there a separate issue surrounding precision that is causing us to skip these tests, or should this PR be fixing these precision issues as well? I don't think i'm fully understanding the above comment